android_cobertura.md

```diff
@@ -0,0 +1,142 @@
+# Configure Android Project To Track Code Coverage With Cobertura
+These are the steps required to setup your Jenkins project to collect code coverage with Covertura, not JaCoCo. The main argument for doing this is to align both iOS and Android projects and have them all report code coverage via Covertura instead of separate plugins. **Note:** We are not eliminating JaCoCo from the Android project. We are using JaCoCo to calculate the actual code coverage, but then converting its results into a format that Covertura can understand for the purposes of reporting the code coverage.
+
+# Jenkins Setup
+
+## coverage.gradle File
```

Comment:
directly link to https://github.com/IntrepidPursuits/AndroidSkeleton/blob/master/app/coverage.gradle so that we have one central place the for coverage.gradle reference